### PR TITLE
add option to seed custom bounding-boxes and filter metro-extracts

### PR DESCRIPTION
**Pull request by sevko**
Increase seeding configurability by:
  * allowing users to filter metro-extracts by city name
  * adding the ability to specify any number of custom bounding boxes

### CODE REVIEW

**Comment by sevko** [comment]
@rmarianski , check this out when you get a chance.

File: config.yaml.sample [hunk 1]
```diff
@@ -21,10 +21,24 @@ tiles:
       url: https://raw.githubusercontent.com/mapzen/metroextractor-cities/master/cities.json
       zoom-start: 11
       zoom-until: 15
+
+      # Can be set to an array of metro-extract city names, like
+      # 'new-york_new-york', to only seed tiles from those areas rather than
+      # all of them.
+      cities: Null
```
Comment (rmarianski) [inline config.yaml.sample]: Can we make this something that is valid yaml? Either: `cities:`, `cities: null`, or `cities: [new-york_new-york]`.

File: tilequeue/command.py [hunk 2]
```diff
@@ -150,7 +157,12 @@ def make_seed_tile_generator(cfg):
     else:
         top_tiles = ()
 
-    combined_tiles = chain(all_tiles, metro_extract_tiles, top_tiles)
+    custom_tiles = tile_generator_for_multiple_bounds(
```
Comment (rmarianski) [inline tilequeue/command.py]: Can this be guarded with an explicit check like the others? Set it to an empty list/tuple if the bounds and/or zooms aren't configured.

File: tilequeue/config.py [hunk 3]
```diff
@@ -102,13 +117,19 @@ def default_yml_config():
...
                     'zoom-start': None,
                     'zoom-until': None,
                 },
-                'should-add-to-tiles-of-interest': True,
+                'custom': {
+                    'zoom-start': None,
+                    'zoom-until': None,
+                    'bboxes': []
+                },
+                'should-add-to-tiles-of-interest': True
```
Comment (rmarianski) [inline tilequeue/config.py]: Good catch!

**Comment by sevko** [comment]
@rmarianski , all addressed.

**Comment by rmarianski** [comment]
:+1:

**Pull request merged**